dashboard/app/controllers/tracking_pixel_controller.rb

```diff
@@ -0,0 +1,22 @@
+require 'cdo/firehose'
+
+class TrackingPixelController < ApplicationController
+  def index
+    prevent_caching
+    curriculum_page = URI.unescape(params[:from]).split('/').reverse.drop(1).reverse.join('/')
+    user_id = current_user&.id || nil
+
+    FirehoseClient.instance.put_record(
+      study: 'curriculum-builder-page-views',
+      study_group: 'v0',
+      event: 'curriculum-builder-page-view',
+      data_json: {
+        curriculumBuilderUrl: curriculum_page,
+        unplugged: curriculum_page.include?("unplugged"),
+        userId: user_id
```

Comment:
I was about to suggest we go ahead and check whether the current user is PD'd here too, but I really like that this route barely uses the DB.  Keeping it simple seems good.